mozdownload/scraper.py

```diff
@@ -516,7 +532,8 @@ def binary_regex(self):
                         'win64': r'(\.installer%(STUB)s)?\.%(EXT)s$'}
         regex = regex_base_name + regex_suffix[self.platform]
 
-        return regex % {'APP': self.application,
+        is_mobile = self.application == 'mobile'
+        return regex % {'APP': 'fennec' if is_mobile else self.application,
```

Comment:
Same here for `not in`. Also I tend to think a mapping would make sense to map from an application name to the folder name on FTP. I would rather have `fennec` as application name than `mobile` given that this is the real application name.